### PR TITLE
Update mysterium-tequilapi

### PR DESCRIPTION
**Pull request by donce**
I need new tequilapi-js for https://github.com/mysteriumnetwork/mysterium-vpn/issues/458

### CODE REVIEW

File: src/renderer/dependencies/modules/provider-service.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ export default function bootstrap (container: Container) {
     'baseURL': 'https://testnet.mysterium.network',
     'serviceType': 'openvpn'
   })
-  container.service('providerService', ['tequilapiClient'], tequilapiClient => {
-    return new ProviderService(tequilapiClient)
+  container.service('providerService', ['tequilapiClient', 'providerConfig'], (tequilapiClient, providerConfig) => {
+    return new ProviderService(tequilapiClient, providerConfig.serviceType)
```
Comment (ignasbernotas) [inline src/renderer/dependencies/modules/provider-service.js]: !! serviceType argument has been removed from the constructor

Comment (donce) [reply src/renderer/dependencies/modules/provider-service.js]: Wtf, why is it back when?

Comment (donce) [reply src/renderer/dependencies/modules/provider-service.js]: Ok, found the reason: https://github.com/mysteriumnetwork/mysterium-vpn-js/pull/10

Comment (donce) [reply src/renderer/dependencies/modules/provider-service.js]: Fixed

**Pull request merged**